lib/tdf3/src/client/builders.ts

```diff
@@ -2,10 +2,15 @@ import { AttributeValidator } from './validation.js';
 import { AttributeObject, Policy } from '../models/index.js';
 import { type RcaParams, type RcaLink, type Metadata } from '../tdf.js';
```

Comment:
Can you move these RCA types and references to virtru-sdk as well? This would include moving some methods and fields from the builders here to their subclasses in virtru-sdk

I think we can do this without requiring a major (or even minor?) version update to virtru-sdk.